README.md

```diff
@@ -15,12 +15,12 @@ Create a workflow `.yml` file in your `.github/workflows` directory. An [example
 
 #### Optional requirement
 Since this action requires user name and password which are sensitive pieces of information, it would be ideal to store them securely. We can achieve this in a GitHub repo by using [secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets). So, click on `settings` tab in your repo and add 2 new secrets:
-- `adminUsername` - store the admin user name in this (e.g. user@contoso.onmicrosoft.com)
-- `adminPassword` - store the password of that user in this.
+- `ADMIN_USERNAME` - store the admin user name in this (e.g. user@contoso.onmicrosoft.com)
```

Comment:
Will this be a breaking change for our users or is it just a change in the docs and we used uppercase already?

Reply:
This is just a documentation change; we suggest to folks who want to use the action to create a secret in GitHub that uses `camelCase`, however if you create a secret in GitHub with `adminUsername` it will be changed to `ADMINUSERNAME`, which maybe confusing, so for consistency, it makes sense for us to use the same casing and format throughout.